src/scripts/Invoke-Task.ps1

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Executes a build task.
+
+    .DESCRIPTION
+    Run this from the /src/scripts folder.
+
+    .PARAMETER Task
+    Required. Name of the task to execute.
+
+    .PARAMETER Silent
+    Optional. Indicates whether Invoke-Build should hide verbose build output. Default = false.
+
+    .PARAMETER Reset
+    Optional. Indicates whether the BuildHelper module should be re-compiled. Use this when changing build tasks and scripts. Default = false.
+    
+    .EXAMPLE
+    ./Invoke-Task -Task Build.PsModule
+
+    Executes the "Build.PsModule" task.
+
+    .EXAMPLE
+    ./Invoke-Task Version
+
+    Executes the "Version" task.
+#>
+param (
+    [Parameter(Mandatory = $true, Position = 0)]
+    [string]
+    $Task,
+
+    [switch]
+    $Major,
+
+    [switch]
+    $Minor,
+
+    [switch]
+    $Patch,
+    
+    [switch]
+    $Prerelease,
+
+    [ValidateSet('dev', 'alpha', 'preview')]
+    [string]
+    $Label = 'dev',
+
+    [switch]
+    $Reset
+)
+
+# If -Reset is specified, remove the BuildHelper module so it can be re-compiled
+if ($Reset) {
+    Remove-Module BuildHelper -ErrorAction SilentlyContinue
```

Comment:
If we want to go back to using scripts I'm fine with it. I can see how them being stuck in a module could add overhead

Reply:
Let's see how many things there are like this. It may be the only one.